scripts/apps/authoring/authoring/index.ts

```diff
@@ -328,7 +328,8 @@ angular.module('superdesk.apps.authoring', [
                     authoringWorkspace: AuthoringWorkspaceService,
                     authoring,
                 ) {
-                    if (appConfig?.corrections_workflow && data.item.state === 'published') {
+                    if (appConfig?.corrections_workflow
+                    && data.item.state === 'published' || data.item.state === 'corrected') {
```

Comment:
If statements don't fit in one line, it'd be easier to read if it was formatted so only one statement was added per line:

```
if (
    a
    && b
    && c
) {
    // ...
}
```